cmd/k8s-operator/nameserver.go

```diff
@@ -0,0 +1,278 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build !plan9
+
+// tailscale-operator provides a way to expose services running in a Kubernetes
```

Comment:
You don't need to repeat this package comment on every file, having it in one place should be sufficient.